packages/design-system/src/styles/components/_Pagination.scss

```diff
@@ -80,9 +95,14 @@ $pagination-page-count-color: $color-gray;
     vertical-align: middle;
     width: 20px;
   }
-
   .ds-c-pagination__nav--image {
     height: 18px;
     width: 18px;
   }
+
+  /* stylelint-disable scss/media-feature-value-dollar-variable */
+  @media (-ms-high-contrast: active), (forced-colors: active) {
+    align-items: center;
```

Comment:
This was added because adding a border to our nav buttons created an unbalanced layout for HCM users.